### ISSUE TITLE
stream.write is not a function

### ISSUE THREAD

**Issue by Leeds-eBooks** [tmcw/sitemap-static#6]
```
/Users/ben/Dev/Git/HireHubStatic/node_modules/sitemap-static/index.js:34
  stream.write(header);
         ^

TypeError: stream.write is not a function
    at module.exports (/Users/ben/Dev/Git/HireHubStatic/node_modules/sitemap-static/index.js:34:10)
    at Object.<anonymous> (/Users/ben/Dev/Git/HireHubStatic/node_modules/sitemap-static/bin/index.js:16:1)
    at Module._compile (module.js:434:26)
    at Object.Module._extensions..js (module.js:452:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:475:10)
    at startup (node.js:117:18)
    at node.js:951:3
```

node version 4.0.0
npm version 2.14.2
sitemap-static version 0.3.0

**Comment by tmcw** [comment]
Ah, https://github.com/tmcw/sitemap-static/commit/1905e2c3701453a9f9d37f8e584f806462192a5a was a bad refactor. fixing.

**Comment by Leeds-eBooks** [comment]
Thanks!

**Comment by tmcw** [comment]
Fixed in 1641b045b80a99d2de46b6db3cef4adc6c09d925 and published as 0.3.1

**Comment by Leeds-eBooks** [comment]
Just confirming it works, cheers!
